src/runtime/manifest.ts

```diff
@@ -63,7 +63,7 @@ export interface ManifestEndpoint {
 
   scheduleTrigger?: {
     schedule?: string;
-    timezone?: string;
+    timeZone?: string;
```

Comment:
hmm this seems like a tricky migration - would changing possibly break users trying to deploy v1 scheduled function w/ CLI that relies/copies `timezone` property instead?

Reply:
Nope, v1 schedule functions use the correct camel case `timeZone` - https://github.com/firebase/firebase-functions/blob/9b6da2d8d12c70914e5ca6258c50b56239e4ad88/src/providers/pubsub.ts#L151

This field wasn't actually directly used in v1 at all. The container contract spec & the CLI both have this being camel case, so this would have been a bug if we left it lowercased

Reply:
Oh this is **fixing* a typo on `manifest.ts` 🤦🏼. 

Sigh I wish TypeScript helped but looks like TS is not going to help w/ situations like this ([playground](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgILIN4Chm+SOAWwgH4AuZAZzClAHMc8xhiAtAexFIutpAYC+WLKEixEKAEKZGuAsXJUa9WcmbEAXp25K+g4Qk7VkcCugC8mNSwgcuFAESV2xa661cHyIQBsIYZAAjCmlLOABuZAB6KPx2ZGgodigSIA)).